test/parallel/test-http-perf_hooks.js

```diff
@@ -57,3 +53,20 @@ server.listen(0, common.mustCall(async () => {
   ]);
   server.close();
 }));
+
+process.on('exit', () => {
+  const httpClients = [];
+  const httpRequests = [];
```

Comment:
Using arrays seems pointless here
```suggestion
  let numberOfHttpClients = 0;
  let numberOfHttpRequests = 0;
```

Reply:
done.